pkg/v1/tkg/web/src/app/app.module.ts

```diff
@@ -1,32 +1,25 @@
 // Angular imports
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
-import { BrowserModule } from '@angular/platform-browser';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-
+import {NgModule} from '@angular/core';
```

Comment:
nit: we're trying to keep imports alphabetized (within the three sections of Angular, 3rd party, app)